cmd/snapshot/main.go

```diff
@@ -63,15 +63,18 @@ func parseBounds(boundString string) {
 		usage()
 		os.Exit(2)
 	}
-	for i, bound := range bounds {
-		FBounds[i], err = strconv.ParseFloat(strings.TrimSpace(bound), 64)
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "Error: invalid bounds provided — %v\n", boundString)
-			fmt.Fprintf(os.Stderr, "Error: unabled to parse %v(%v) as a float.\n", bound, i)
-			usage()
-			os.Exit(2)
-		}
+
+	y1, err := strconv.ParseFloat(strings.TrimSpace(bounds[0]), 64)
+	x1, err := strconv.ParseFloat(strings.TrimSpace(bounds[1]), 64)
+	y2, err := strconv.ParseFloat(strings.TrimSpace(bounds[2]), 64)
+	x2, err := strconv.ParseFloat(strings.TrimSpace(bounds[3]), 64)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error: invalid bounds provided — %v\n", boundString)
+		//fmt.Fprintf(os.Stderr, "Error: unabled to parse %v(%v) as a float.\n", )
```

Comment:
remove commented out code.

Reply:
You are ignoring all the errors for y1,x1, and y2. Here you need to duplicate the err check after each one.  If you are want to do it this way instead of using the for loop.